plugins/social_network/social_network.cpp

```diff
@@ -609,20 +724,27 @@ namespace golos { namespace plugins { namespace social_network {
         });
     }
 
-    void fill_comment_content(const golos::chain::database& db, const comment_object& co, comment_api_object& con) {
-        if (!db.has_index<comment_content_index>()) {
-            return;
-        }
-        const auto content = db.find<comment_content_object, by_comment>(co.id);
-        if (content != nullptr) {
-            con.title = to_string(content->title);
-            con.body = to_string(content->body);
-            con.json_metadata = to_string(content->json_metadata);
+    void fill_comment_info(const golos::chain::database& db, const comment_object& co, comment_api_object& con) {
+        if (db.has_index<comment_content_index>()) {
+            const auto content = db.find<comment_content_object, by_comment>(co.id);
+            if (content != nullptr) {
+                con.title = to_string(content->title);
+                con.body = to_string(content->body);
+                con.json_metadata = to_string(content->json_metadata);
+            }
+
+            const auto root_content = db.find<comment_content_object, by_comment>(co.root_comment);
+            if (root_content != nullptr) {
+                con.root_title = to_string(root_content->title);
+            }
         }
 
-        const auto root_content = db.find<comment_content_object, by_comment>(co.root_comment);
-        if (root_content != nullptr) {
-            con.root_title = to_string(root_content->title);
+        if (db.has_index<comment_last_update_index>()) {
+            const auto last_update = db.find<comment_last_update_object, by_comment>(co.id);
+            if (last_update != nullptr) {
+                con.active = last_update->active;
+                con.last_update = last_update->last_update;
```

Comment:
`active` and `last_update` is optional now.
Set value to `time_point_sec::min()` if `last_update == nullptr`